.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/fbidu/miniconda3/envs/python38/bin/python"
+}
```

Comment:
I think you didn't want to commit that.
Suggestion:
```
git config --global core.excludesfile ~/.gitignore_global
echo ".vscode" > ~/.gitignore_global
```

Reply:
Oh, sorry! And thanks for the tip :sweat_smile: